pyproject.toml

```diff
@@ -43,6 +43,7 @@ snowflake = ["snowflake-connector-python[pandas]"]
 mysql = ["mysql-connector-python"]
 postgres = ["psycopg2-binary"]
 server = ["fastapi", "httpx", "uvicorn"]
+boto3 = ["boto3"]
```

Comment:
This should be part of the `dev-dependencies`.
It's only used in testing, not related to any PyDough functionality.